### DISCUSSION TITLE
希望增加json转换excel功能，并支持下载转换后的excel

### DISCUSSION THREAD

**Discussion by gljack** [he3-app/he3-feedback#342] [Feature Requests]
### One sentence introduction the new tool or features

增加json转换excel功能，并支持下载转换后的excel

## Description of new tools or features
希望增加json转换excel功能
支持[{}]数组类型转换excel，只需要转换第一层
## Scenarios for use
现有数据大部分基于json交互，但是数据统计的时候转换成excel更方便。目前本人常用操作是先将json通过json Path获取到指定的数组json，再通过其他json转换excel工具变为excel做数据分析
## Link or screenshot
<img width="931" alt="image" src="https://github.com/he3-app/he3-feedback/assets/19208641/807cdc55-4f39-4332-a8cb-809633486d77">
https://wejson.cn/json2excel/

### System you are using

macOS intel

**Comment by zhenweiwang1990** [comment]
有 json 转 csv 的，csv 可以直接用 excel 打开。
